os-config.tpl.yml

```diff
@@ -150,6 +150,7 @@ rancher:
       - /usr/bin/iptables:/sbin/iptables:ro
       - /media:/media:shared
       - /mnt:/mnt:shared
+      - /:/host
```

Comment:
But isn't being run in the installer container and not the console? `/dev` in the console container should be the same as `/dev` on the host anyways.